docs/analysis/data-cleaning.md

```diff
@@ -0,0 +1,81 @@
+Data Cleaning
+=============
+
+This module is to clean the extracted sensor features before merging it with the target labels.
+    
+## Clean sensor features for individual participants
+
+!!! info "File Sequence"
+    ```bash
+    - data/processed/features/{pid}/all_sensor_features.csv
+    - data/processed/features/{pid}/all_sensor_features_cleaned_{provider_key}.csv
+    ```
+
+### RAPIDS provider
+
+Parameters description for `[ALL_CLEANING_INDIVIDUAL][PROVIDERS][RAPIDS]`:
+
+|Key&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;            | Description |
+|----------------|-----------------------------------------------------------------------------------------------------------------------------------
+|`[COMPUTE]` | Set to `True` to clean sensor features for individual participants from the `RAPIDS` provider|
+|`[IMPUTE_SELECTED_EVENT_FEATURES]`     | Fill NA with 0 for the selected event features, see table below
+|`[COLS_NAN_THRESHOLD]`                 | Discard columns with missing value ratios higher than `[COLS_NAN_THRESHOLD]`. Set to 1 to disable
+|`[COLS_VAR_THRESHOLD]`                 | Set to `True` to discard columns with zero variance
+|`[ROWS_NAN_THRESHOLD]`                 | Discard rows with missing value ratios higher than `[ROWS_NAN_THRESHOLD]`. Set to 1 to disable
+|`[DATA_YIELDED_HOURS_RATIO_THRESHOLD]` | Discard rows with `phone_data_yield_rapids_ratiovalidyieldedhours` feature less than `[DATA_YIELDED_HOURS_RATIO_THRESHOLD]`. Set to 0 to disable
+|`DROP_HIGHLY_CORRELATED_FEATURES`      | Discard highly correlated features, see table below
+
+Parameters description for `[ALL_CLEANING_INDIVIDUAL][PROVIDERS][RAPIDS][IMPUTE_SELECTED_EVENT_FEATURES]`:
+
+|Parameters                             | Description                                                    |
+|-------------------------------------- |----------------------------------------------------------------|
+|`[COMPUTE]`                            | Set to `True` to fill NA with 0 for the selected event features
```

Comment:
Set to `True` to fill NAs with 0 for phone event-based features

Reply:
We need to include the features that we are imputing in this step (maybe an expandable section?)

Reply:
Hi @JulioV, I am not sure how to list the selected event-based features as we use [regular expressions](https://github.com/carissalow/rapids/blob/data_cleaning/src/features/all_cleaning_individual/rapids/main.R#L18) to match these features. How about adding the following descriptions after [line49](https://github.com/carissalow/rapids/blob/data_cleaning/docs/analysis/data-cleaning.md?plain=1#L49)?

```
The following phone event-based features are considered currently:
- Application foreground features: countevent, countepisode, minduration, maxduration, meanduration, sumduration.
- Battery features: all features.
- Calls features: count, distinctcontacts, sumduration, minduration, maxduration, meanduration, modeduration.
- Keyboard features: sessioncount, averagesessionlength, changeintextlengthlessthanminusone, changeintextlengthequaltominusone, changeintextlengthequaltoone, changeintextlengthmorethanone, maxtextlength, totalkeyboardtouches.
- Messages features: count, distinctcontacts.
- Screen features: sumduration, maxduration, minduration, avgduration, countepisode.
- WiFi features: all connected and visible features.

Reply:
Yep, this works